alcs-frontend/src/app/shared/details-header/details-header.component.ts

```diff
@@ -25,7 +25,7 @@ export class DetailsHeaderComponent {
 
   linkedCards: (CardDto & { displayName: string })[] = [];
 
-  @Input() heading = 'Title Here';
+  @Input() heading = 'Application';
```

Comment:
This was intentional set here as an obvious placeholder. A more correct fix would be to set this input property in the HTML file where it's used.